tests/test_parser.py

```diff
@@ -400,6 +400,11 @@ def test_master_playlist_with_frame_rate():
     assert 60 == playlists_list[2]['stream_info']['frame_rate']
     assert 12.5 == playlists_list[3]['stream_info']['frame_rate']
 
+def test_master_playlist_with_unrounded_frame_rate():
+    data = m3u8.parse(playlists.VARIANT_PLAYLIST_WITH_ROUNDABLE_FRAME_RATE)
+    playlists_list = list(data['playlists'])
+    assert 12.54321 == playlists_list[0]['stream_info']['frame_rate']
```

Comment:
Shouldn't be already fixed 3 digits here?

```python
     assert 12.543 == playlists_list[0]['stream_info']['frame_rate'] 
```

Reply:
This verifies we can parse a playlist containing a frame rate longer than 3 digits - it isn't rounded until it is converted back to string with dumps